src/auth/token-verifier.ts

```diff
@@ -177,15 +179,20 @@ export interface DecodedIdToken {
 // Audience to use for Firebase Auth Custom tokens
 const FIREBASE_AUDIENCE = 'https://identitytoolkit.googleapis.com/google.identity.identitytoolkit.v1.IdentityToolkit';
 
-export const ALGORITHM_RS256 = 'RS256';
-
 // URL containing the public keys for the Google certs (whose private keys are used to sign Firebase
 // Auth ID tokens)
 const CLIENT_CERT_URL = 'https://www.googleapis.com/robot/v1/metadata/x509/securetoken@system.gserviceaccount.com';
 
 // URL containing the public keys for Firebase session cookies. This will be updated to a different URL soon.
 const SESSION_COOKIE_CERT_URL = 'https://www.googleapis.com/identitytoolkit/v3/relyingparty/publicKeys';
 
+/**
+ * User facing token information related to the Firebase ID token.
+ *
+ * @internal
+ */
```

Comment:
This looks off. This comment shouldn't be here.

Reply:
Thanks. That looks like another bad merge. Removed.